src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -163,20 +164,20 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
         return (
           <Box>
             <Join separator={<Spacer y={[4, 6]} />}>
-              <Input
-                title="Alert Name"
-                name="name"
-                placeholder={
-                  isFallbackToGeneratedAlertNamesEnabled
-                    ? undefined
-                    : entity.placeholder
-                }
-                value={values.name}
-                onChange={handleChange("name")}
-                onBlur={handleBlur("name")}
-                error={errors.name}
-                maxLength={75}
-              />
+              {isFallbackToGeneratedAlertNamesEnabled ? (
+                <SavedSearchAlertNameInputQueryRenderer />
+              ) : (
```

Comment:
Might be better to just use the new component here (as we [did](https://github.com/artsy/force/pull/12751/files#diff-5b7f8a8cb9b3b800721b3117460dd2172c5525dbc121b85fad4aa6ec17fe8542R139) in the create modal) instead of branching based on the ff.  But doing so caused some test failures that I didn't have time to resolve. We can either try to address that before merging, or update the tests when we remove the branching.

Reply:
I guess, it's okay to leave it like this and clean up the code when we remove the following.